docs/automated-testing/best-practices-for-writing-acceptance-tests.md

```diff
@@ -0,0 +1,26 @@
+# Best Practices for Writing Acceptance Tests
```

Comment:
shouldn't be link to this file present in appropriate `index.md` and navigation.yml files?

Reply:
a181074fe6148f5c056dc2abe979acd0da489547